src/Evolution/Systems/NewtonianEuler/BoundaryConditions/BoundaryCondition.hpp

```diff
@@ -0,0 +1,37 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pup.h>
+
+#include "Domain/BoundaryConditions/BoundaryCondition.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace NewtonianEuler::BoundaryConditions {
+template <size_t Dim>
+class DirichletAnalytic;
+}  // namespace NewtonianEuler::BoundaryConditions
+/// \endcond
+
+/// \brief Boundary conditions for the scalar wave system
+namespace NewtonianEuler::BoundaryConditions {
+/// \brief The base class off of which all boundary conditions must inherit
+template <size_t Dim>
+class BoundaryCondition : public domain::BoundaryConditions::BoundaryCondition {
+ public:
+  using periodic = void;
```

Comment:
Huh? I see you remove it in the next commit, but should it be removed here?

Reply:
Yea, can be removed in the first commit. Squashed that in